src/Common.props

```diff
@@ -0,0 +1,55 @@
+<Project>
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <RuntimeIdentifiers>win-x64;win-x86;linux-x64;linux-arm;rhel.6-x64;osx-x64</RuntimeIdentifiers>
+    <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
+    <AssetTargetFallback>portable-net45+win8</AssetTargetFallback>
+    <NoWarn>NU1701;NU1603;</NoWarn>
+    <OutputType>OVERRIDE_ME</OutputType>
+    <OSPlatform>OS_UNKNOWN</OSPlatform>
+    <OSArchitecture>ARCH_UNKNOWN</OSArchitecture>
+    <DebugConstant></DebugConstant>
+    <VssApiVersion>0.5.125-private</VssApiVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <DebugType>portable</DebugType>
+    <DebugConstant>DEBUG</DebugConstant>
+
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Windows)))' == 'true'">
+    <OSPlatform>OS_WINDOWS</OSPlatform>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::OSX)))' == 'true'">
+    <OSPlatform>OS_OSX</OSPlatform>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Linux)))' == 'true'">
+    <OSPlatform>OS_LINUX</OSPlatform>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(OSPlatform)' == 'OS_WINDOWS' AND '$(PROCESSOR_ARCHITECTURE)' == 'AMD64'">
+    <OSArchitecture>X64</OSArchitecture>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(OSPlatform)' == 'OS_WINDOWS' AND '$(PROCESSOR_ARCHITECTURE)' == 'X86'">
+    <OSArchitecture>X86</OSArchitecture>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(OSPlatform)' == 'OS_OSX'">
+    <OSArchitecture>X64</OSArchitecture>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(OSPlatform)' == 'OS_LINUX' AND '$(PackageRuntime)' == 'linux-x64'">
+    <OSArchitecture>X64</OSArchitecture>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(OSPlatform)' == 'OS_LINUX' AND '$(PackageRuntime)' == 'rhel.6-x64'">
+    <OSArchitecture>OS_RHEL6;X64</OSArchitecture>
```

Comment:
will this cause OS_LINUX;OS_RHEL6 both get defined?

Reply:
I can't explain it, but I also didn't change it :)
https://github.com/microsoft/azure-pipelines-agent/blob/90dff0213a33cf484594c30420c0f1aa3085c6f6/src/Microsoft.VisualStudio.Services.Agent/Microsoft.VisualStudio.Services.Agent.csproj#L53